master/internal/api_model.go

```diff
@@ -469,7 +496,7 @@ func (a *apiServer) PatchModelVersion(
 
 	finalModelVersion := &modelv1.ModelVersion{}
 	err = a.m.db.QueryProto("update_model_version", finalModelVersion, req.ModelVersionId,
-		parentModel.Model.Id, currModelVersion.Name, currModelVersion.Comment, currModelVersion.Notes,
+		parentModel.Id, currModelVersion.Name, currModelVersion.Comment, currModelVersion.Notes,
```

Comment:
I don't understand what the point of this change is.  It looks like we haven't modified the proto so it seems like this shouldn't be changed, unless it was broken before.

Reply:
I changed from `parentModel := a.GetModel` (where Model was inside of the returned GetModelResponse object) to `parentModel := currModelVersion.Model`

Reply:
ah, this makes sense